### PR TITLE
rpi-imager: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/rpi-imager.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# rpi-imager
+
+> Flash images onto storage drives.
```
Comment (mfrw) [inline pages/common/rpi-imager.md:3]: Should we say devices instead of drives ?
```suggestion
> Flash images onto storage devices.
```

**Review by mfrw — COMMENTED** [review]
Thanks @CleanMachine1 
A teeny nit, but LGTM :)

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by mfrw — APPROVED** [review]
LGTM

**Pull request merged**